plutus-conformance/test-cases/uplc/evaluation/term/lam/lam-2/lam-2.uplc

```diff
@@ -0,0 +1,4 @@
+-- From  https://github.com/HachiSecurity/plc-llvm/tree/main/compiler/test-data/untyped
```

Comment:
Not sure if we should add these comments in the tests. See my other comment.

Reply:
I'll remove all of those.